src/storage/src/hummock/sstable/builder.rs

```diff
@@ -52,6 +53,7 @@ impl Default for SSTableBuilderOptions {
     }
 }
 
+pub const BITMAP_LEN: usize = 1 << ((VALUE_META_SIZE << 3) - 3);
```

Comment:
How is this computed? What's stored in `bitmap` (the mapping relationship)?

Reply:
VALUE_META_SIZE << 3: value meta size (bits)
1<< (VALUE_META_SIZE << 3): bitmap for vnode -> {true/false} (bits)

Reply:
> How is this computed? What's stored in `bitmap` (the mapping relationship)?

VALUE_META_SIZE << 3: value meta size (bits)
1<< (VALUE_META_SIZE << 3): bitmap for vnode -> {true/false} (bits)